src/systemtest/resources/30daysFromUntilDate/expected/summary.json

```diff
@@ -1 +1 @@
+{"reportGeneratedTime":"Tue Jul 24 17:45:15 SGT 2018","reportGenerationTime":"15 second(s)","zoneId":"Asia/Singapore","reportTitle":"RepoSense Report Test Title","repos":[{"location":{"location":"https://github.com/reposense/testrepo-Alpha.git","repoName":"testrepo-Alpha","organization":"reposense"},"branch":"master","displayName":"reposense/testrepo-Alpha[master]","outputFolderName":"reposense_testrepo-Alpha_master"},{"location":{"location":"https://github.com/reposense/testrepo-Beta.git","repoName":"testrepo-Beta","organization":"reposense"},"branch":"master","displayName":"reposense/testrepo-Beta[master]","outputFolderName":"reposense_testrepo-Beta_master"},{"location":{"location":"https://github.com/reposense/testrepo-Charlie.git","repoName":"testrepo-Charlie","organization":"reposense"},"branch":"master","displayName":"reposense/testrepo-Charlie[master]","outputFolderName":"reposense_testrepo-Charlie_master"},{"location":{"location":"https://github.com/reposense/testrepo-Delta.git","repoName":"testrepo-Delta","organization":"reposense"},"branch":"master","displayName":"reposense/testrepo-Delta[master]","outputFolderName":"reposense_testrepo-Delta_master"}],"errorSet":[{"repoName":"--github.com-reposense/testrepo-Beta[HEAD]","errorMessage":"Failed to clone from ttps://github.com/reposense/testrepo-Beta.git"},{"repoName":"reposense/testrepo-Delta[nonExistentBranch]","errorMessage":"Branch \"nonExistentBranch\" does not exist."}],"sinceDate":"2017-10-01","untilDate":"2017-11-01","isSinceDateProvided":false,"isUntilDateProvided":true}
```

Comment:
I think the error message is not correct. We should retain the fact that `ttps://github.com/reposense/testrepo-Beta.git` is an invalid URL (because Git only supports ssh, git, http, https, ftp and ftps protocols). Giving the message that it failed to clone is not specific enough, and the user will need to take an extra step to figure out why.

Reply:
I believe the behaviour here is that this URL is instead detected as part of the scp-like protocol for cloning (similar to the `git@github.com:...`. In this case it is the host name `ttps` with path `//github.com/reposense/testrepo-Beta.git` which is a valid pathname as multiple slashes are just treated as a single slash.